src/pybind/mgr/dashboard/services/rgw_client.py

```diff
@@ -1648,21 +1631,3 @@ def get_primary_zonedata(self, data):
             return match.group(1)
 
         return ''
-
-    def get_shards_info(self, data):
```

Comment:
Is there a chance we need this info in future? @aaSharma14 
Same question for `get_behind_shards`

Reply:
We are showing this info in the sync status popover..I don't think any duplicate info will be useful in future as well